src/form/Form.jsx

```diff
@@ -532,7 +532,10 @@ function Form({
           stepChanged = changeStep(stepKey, newKey, steps, history);
         },
         firstStepLoaded: first,
-        integrationData
+        integrationData,
+        stepStyle: {
```

Comment:
i think styles: { step: { backgroundColor: ... }  }  would be a little more encompassing, and you can probably include this in commonCallbackProps so it's available to every callback function. also this will need to be added to the sdk documentation too